Error occurred while {{ action }}:

{{ exception }}

Traceback:
{% for line in traceback %}
{{ line }}{% endfor %}